tools/README.md

```diff
@@ -10,10 +10,9 @@ openpilot is developed and tested on **Ubuntu 20.04**, which is the primary deve
 First, clone openpilot:
 ``` bash
 cd ~
-git clone https://github.com/commaai/openpilot.git
+git clone --recursive https://github.com/commaai/openpilot.git
 
-cd openpilot 
-git submodule update --init
```

Comment:
can you keep this and add a comment like "openpilot uses git submodules, update them with:"? we should be better about including all the common workflow stuff here